#- Adds a pkgconfig/bzip2.pc file 
#
# author: Jiri Furst <jiri.furst@gmail.com>
# inspired by OpenSUSE patch by Stanislav Brabec <sbrabec@suse.cz>, see
# http://ftp.suse.com/pub/people/sbrabec/bzip2/
diff -Nau bzip2-1.0.6.orig/bzip2.pc.in bzip2-1.0.6/bzip2.pc.in
--- bzip2-1.0.6.orig/bzip2.pc.in	1970-01-01 01:00:00.000000000 +0100
+++ bzip2-1.0.6/bzip2.pc.in	2019-05-01 11:47:29.795517973 +0200
@@ -0,0 +1,11 @@
+exec_prefix=${prefix}
+bindir=${exec_prefix}/bin
+libdir=${exec_prefix}/lib
+includedir=${prefix}/include
+
+Name: bzip2
+Description: Lossless, block-sorting data compression
+Version: 1.0.6
+Libs: -L${libdir} -lbz2
+Cflags: -I${includedir}
+
diff -Nau bzip2-1.0.6.orig/Makefile bzip2-1.0.6/Makefile
--- bzip2-1.0.6.orig/Makefile	2019-05-01 11:28:04.788206974 +0200
+++ bzip2-1.0.6/Makefile	2019-05-01 11:46:20.911324226 +0200
@@ -107,6 +107,9 @@
 	echo ".so man1/bzgrep.1" > $(PREFIX)/man/man1/bzfgrep.1
 	echo ".so man1/bzmore.1" > $(PREFIX)/man/man1/bzless.1
 	echo ".so man1/bzdiff.1" > $(PREFIX)/man/man1/bzcmp.1
+	if ( test ! -d $(PREFIX)/lib/pkgconfig ) ; then mkdir -p $(PREFIX)/lib/pkgconfig ; fi
+	echo "prefix=$(PREFIX)" > $(PREFIX)/lib/pkgconfig/bzip2.pc
+	cat bzip2.pc.in >> $(PREFIX)/lib/pkgconfig/bzip2.pc
 
 clean: 
 	rm -f *.o libbz2.a bzip2 bzip2recover \
